docs: Comprehensive update to admin documentation

randybias opened the pull request:
- Major revisions to multicluster service documentation
- Significant updates to backup documentation
- Updates to cluster creation procedures
- Minor improvements to credentials and hosted control planes
- Revised service templates documentation
- Updated AWS VPC troubleshooting guide

a13x5 commented on docs/template/main.md:
```diff
@@ -1,24 +1,37 @@
+(`metadata.namespace`).
 
-Here is an example of a custom `Template` with the `HelmChart` reference:
+> Note. `ClusterTemplate` and `ServiceTemplate` objects should reside in the same namespace as the `ManagedCluster`
```
Incorrect syntax for notes, because of mkdocs stuff.
Should be `> NOTE:` on a separate line

eromanova replied:
Oh, sorry. I've missed that.

a13x5 reviewed (changes requested):
One minor note regarding call-out syntax.
Rest of it looks good

randybias committed:
docs: Comprehensive update to admin documentation

- Major revisions to multicluster service documentation
- Significant updates to backup documentation
- Updates to cluster creation procedures
- Minor improvements to credentials and hosted control planes
- Revised service templates documentation
- Updated AWS VPC troubleshooting guide
